releases/1.6/edge/kubeflow/bundle.yaml

```
@@ -0,0 +1,202 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook:
+    charm: admission-webhook
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: admission-webhook-operator
+  argo-controller:
+    charm: argo-controller
+    channel: 3.3/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  argo-server:
+    charm: argo-server
+    channel: 3.3/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  dex-auth:
+    charm: dex-auth
+    channel: 2.31/edge
+    scale: 1
+    trust: true
+    _github_repo_name: dex-auth-operator
+  envoy:
+    charm: envoy
+    channel: 1.20/edge
+    scale: 1
+    _github_repo_name: envoy-operator
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+  istio-pilot:
+    charm: istio-pilot
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+    options:
+      default-gateway: kubeflow-gateway
+  jupyter-controller:
+    charm: jupyter-controller
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: notebook-operators
+  jupyter-ui:
+    charm: jupyter-ui
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: notebook-operators
+  katib-controller:
+    charm: katib-controller
+    channel: 0.14/edge
+    scale: 1
+    _github_repo_name: katib-operators
+  katib-db:
+    charm: charmed-osm-mariadb-k8s
+    channel: latest/edge
```

This shouldn't matter, but in past we've pinned to stable even in edge bundle
```suggestion
    channel: latest/stable
```